config.microsoftInternal.dsc

```diff
@@ -49,6 +49,9 @@ export const pkgs = isMicrosoftInternal ? [
     { id: "PowerShell.Core", version: "6.1.0" },
     { id: "Dotnet-Runtime", version: "2.2.2.1" },
 
+    // VM
+    { id: "CloudBuild.VmCommandProxy", version: "19.4.28.1036" },
+

```

Comment:
Would you mind moving it a bit higher and keep all cloudbuild related packages together. i.e. next to `CB.QTest`?